src/screens/TransactionConfirmationScreen.js

```diff
@@ -622,6 +622,7 @@ export default function TransactionConfirmationScreen() {
       ...gasParams,
       nonce,
     };
+    delete txPayloadUpdated?.chainId;
```

Comment:
shouldn't we add it in `omit` before https://github.com/rainbow-me/rainbow/pull/3593/files#diff-f2aec906d3575ac26f0daf12a43661c34a7e0b1a4cdc082ca4c56ec0ad7bf404R615 ?

Reply:
this sounds cleaner ill change

Reply:
actually not gonna do this since we're in the process of removing lodash function calls

Reply:
don't want to block it but the PR still in review https://github.com/rainbow-me/rainbow/pull/3351 and we'll have to migrate this omit anyways, using `omitFlatten` not `delete`